utils/utils.go

```diff
@@ -225,7 +225,14 @@ func UnmarshalFromDir(path string) (map[string]interface{}, error) {
 	}
 	app[servicesName] = services
 
-	environments, err := unmarshalJSONFilesWithFilenames(filepath.Join(path, environmentsName))
+	environmentsPath := filepath.Join(path, environmentsName)
+	_, err = os.Stat(environmentsPath)
+	if err != nil {
```

Comment:
i'm thinking we probably to invert this:

```go
environmentsPath := filepath.Join(path, environmentsName)
_, err = os.Stat(environmentsPath)
if err == nil {
  environments, err := unmarshalJSONFilesWithFilenames(environmentsPath)
  // ...
}
```

that way future devs won't have to know to make any additions here above the `environments` processing (hopefully we're not adding more code ever, but I think just with the way this whole function is constructed, it makes sense to go this route)

Reply:
Sure, that makes sense I'll do that.

1.